argoverse/evaluation/stereo/utils.py

```diff
@@ -67,6 +67,7 @@ def compute_disparity_error(
     density = num_pixels_all_est / num_pixels_all
 
     if density < 1.0:
+        pred_disparity[pred_disparity == 0] = -1  # Set invalid disparities to -1
```

Comment:
Was this previously done in the `.pyx` file?

Reply:
Yep, I just moved it.

Reply:
Put it into the disparity interpolation function. Looks better.